beacon-chain/rpc/eth/validator/handlers_block.go

```diff
@@ -298,12 +299,21 @@ func getConsensusBlockValue(ctx context.Context, blockRewardsFetcher rewards.Blo
 		// ignore for phase 0
 		return "", nil
 	}
-	//get consensus payload value which is the same as the total from the block rewards api
+	// Get consensus payload value which is the same as the total from the block rewards api.
+	// The value is in Gwei, but Wei should be returned from the endpoint.
 	blockRewards, httpError := blockRewardsFetcher.GetBlockRewardsData(ctx, bb)
```

Comment:
can you add a comment for GetBlockRewardsData that the rewards are in wei for the public function?